src/sensor/ping360.cpp

```diff
@@ -151,28 +155,28 @@ void Ping360::handleMessage(const ping_message& msg)
 {
     qCDebug(PING_PROTOCOL_PING360) << "Handling Message:" << msg.message_id();
 
-    // Update frequency for each
-    messageFrequencies[msg.message_id()].setElapsed(_messageElapsedTimer.elapsed());
-    // Since we don't have a huge number of messages and this variable is pretty simple,
-    // we can use a single signal to update someone about the frequency update
-    emit messageFrequencyChanged();
-
     switch (msg.message_id()) {
 
     case CommonId::DEVICE_INFORMATION: {
         if(_configuring) {
             _baudrateConfigurationTimer.start();
             checkBaudrateProcess();
-            return;
         } else {
             _baudrateConfigurationTimer.stop();
             _timeoutProfileMessage.start();
             requestNextProfile();
-            return;
         }
+        return;
     }
 
     case Ping360Id::DEVICE_DATA: {
+        // Request next message ASAP
+        // request another transmission
+        requestNextProfile();
```

Comment:
I thought we needed to extract the current device angle from the device_data message before calling this.

Reply:
Done.